os/pm/pm_initialize.c

```diff
@@ -81,6 +81,26 @@ struct pm_global_s g_pmglobals;
  * Public Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: pm_start
+ *
+ * Description:
+ *   This function is called by the application thread to start the Power
+ *   Management system. This fucntion sets the ready_to_state_change flag which
+ *   enables pm to transition between low and high power states.
+ *
+ * Input parameters:
+ *   None.
+ *
+ * Returned value:
+ *    None.
+ *
+ ****************************************************************************/
```

Comment:
Also it seems, If any one domain/platform user once starts this will enable the pm. Is this our intension? Also once set there is no provision to clear this. Any reason for this

Reply:
Our intention is to give product team an API, which they can call to start PM after `START TASK MAIN`